packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/UPGRADE.txt

```diff
@@ -0,0 +1 @@
+For upgrade instructions, please see http://neo4j.com/docs/operations-manual/current/#upgrading.
```

Comment:
Will it make more sense to provide upgrade link for specific version instead of current?